workspace/__lib__/xod-dev/w5500/http-request(inet)/patch.xodp

```diff
@@ -290,7 +290,7 @@
       "boundLiterals": {
         "__out__": "80"
       },
-      "description": "TCP port number on which the server is listening",
+      "description": "The TCP port number on which the server is listening",
```

Comment:
A port number on which the server is listening. In most configurations, HTTP web servers listen at port 80.